frontend/src/app/modules/external-booking/external-booking-choose-members-page/external-booking-choose-members-page.component.sass

```diff
@@ -0,0 +1,105 @@
+@import '../../../../styles/color-variables'
+@import '../../../../styles/screen-variables'
+
+h1, h3, h4
+    font-family: 'Inter'
+    font-style: normal
+    line-height: 20px
+    color: $pale-sky
+
+h3, h4
+    font-weight: 400
+    font-size: 14px
+
+h1
+    font-weight: 700
+    font-size: 20px
+
+.member-cards-container
+    display: inline-block
+    width: 40%
+    margin-right: 10%
+    margin-bottom: 3%
+
+::ng-deep .team-member-card.mat-card
+    width: 100% !important
+    height: 37px !important
+    margin: 5px 20px !important
+    padding-top: 20px !important
+    padding-left: 20px !important
```

Comment:
Are you sure we need !important here?

Reply:
Yes. Any style of other pages has not been modified:
![image](https://user-images.githubusercontent.com/71783287/186858972-9cc51f01-0e34-4949-a3f6-96025fcf52d9.png)
![image](https://user-images.githubusercontent.com/71783287/186859006-f15c008a-a548-40f4-ad6a-25beefe840f6.png)

Reply:
I mean wouldn't it work without !important?

Reply:
It works. I deleted '!important' labels.